packages/react/src/components/Authenticator/RouteContainer/types.ts

```diff
@@ -1,6 +1,6 @@
 export type RouteProps = {
-  className: string;
-  variation: 'default' | 'modal';
+  className?: string;
+  variation?: 'default' | 'modal';
```

Comment:
These are being passed from the Authenticator and the AuthenticatorProps mark these as optional.